stdlib/camlinternalMod.ml

```diff
@@ -13,7 +13,7 @@
 (*                                                                        *)
 (**************************************************************************)
 
-external make_forward : Obj.t -> Obj.t -> unit = "caml_obj_make_forward"
+external _make_forward : Obj.t -> Obj.t -> unit = "caml_obj_make_forward"
```

Comment:
If this is no longer used, please remove the declaration, don't stick an underscore in front to silence the warning.